compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -830,8 +829,8 @@ object SymDenotations {
             i"""
                | Access to protected $this not permitted because enclosing ${ctx.owner.enclosingClass.showLocated}
                | is not a subclass of ${owner.showLocated} where target is defined""")
-        else if (
-          !(  isType // allow accesses to types from arbitrary subclasses fixes #4737
+        else if
+          (!(  isType // allow accesses to types from arbitrary subclasses fixes #4737
```

Comment:
The formatting broke the alignment of isType with the conditions below, if there's no way to preserve it then the spaces in front of isType should be removed at least.